package/MDAnalysis/lib/qcprot.pyx

```diff
@@ -240,6 +246,41 @@ def InnerProduct(np.ndarray[np.float64_t,ndim=1] A,
 
 @cython.boundscheck(False)
 @cython.wraparound(False)
+def CalcRMSDRotationalMatrix(np.ndarray[np.float64_t,ndim=2] ref,
+                             np.ndarray[np.float64_t,ndim=2] conf,
+                             int N,
+                             np.ndarray[np.float64_t,ndim=1] rot,
+                             np.ndarray[np.float64_t,ndim=1] weights):
+    """
+    Calculate the RMSD & rotational matrix.
+
+    Parameters
+    ----------
+    ref : ndarray, np.float64_t
+        reference structure coordinates
+    conf : ndarray, np.float64_t
+        condidate structure coordinates
+    N : int
+        size of the system
+    rot : ndarray, np.float64_t
+        array to store rotation matrix. Must be flat
+    weights : ndarray, npfloat64_t (optional)
+        weights for each component
+
+    Returns
+    -------
+    rmsd : float
+        RMSD value
+
+    .. versionchanged:: 0.16.0
+       Changed array shape from 3xN to Nx3.
+    """
+    cdef double E0
+    cdef np.ndarray[np.float64_t,ndim=1] A = np.zeros(9,dtype = np.float64)
```

Comment:
@rbrtdlgd – have a look at [PEP8](https://www.python.org/dev/peps/pep-0008/).

In this case, spaces after commas. But read PEP8.